schema.graphql

```diff
@@ -166,6 +196,9 @@ type Query {
   """Get all users"""
   allUsers: [User!]
 
+  """Get all courses preferences"""
+  coursePreferences: CoursePreference!
```

Comment:
this should return a _list_ of course preferences, not a single one, since it returns all courses preferences

Reply:
Corrected, thanks!